ledger/notary-plugin/src/main/kotlin/net/corda/v5/ledger/notary/plugin/core/NotaryError.kt

```diff
@@ -0,0 +1,3 @@
+package net.corda.v5.ledger.notary.plugin.core
+
+interface NotaryError
```

Comment:
This should probably have a kdoc comment to explain what it is / how it is used, given that someone looking just at the `corda-api` repo will have no further context beyond this file.